### DISCUSSION TITLE
dificuldade em entender este processo de divulgar um app

### DISCUSSION THREAD

**Discussion by Betezek** [community/community#70070] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Prezados, estou estudando um clone doi uber e não consigo divulgar no ghitub. Sou novo por aqui e gostaria de uma explicação ouaté uma ajudinha.
BrasilDrivers
meu projeto.
tem como me orientar...

**Comment by DJLink** [comment]
Divulgar em que sentido? Mais pessoas saberem dele?

**Comment by Betezek** [comment]
Colocar ele na rede.Fazer com que mais pessoas vejam.Em 14 de out. de 2023 00:18, David Amador ***@***.***> escreveu:
Divulgar em que sentido? Mais pessoas saberem dele?

**Reply by DJLink** [reply]
O GitHub não serve como meio publicitário, em termos de fazer com que mais pessoas vejam é usar redes sociais, blogs, forums etc.

**Comment by Betezek** [comment]
Prezado, acho que sua linguagem não entendeu o que significa divulgar um app...
Não é divulgar, fazer publicidade, ou o que pensou. É simplesmente fazer ele rodar na rede, entrar no ar, ser visualizado..



Enviado do Email para Windows

De: David Amador
Enviado:domingo, 15 de outubro de 2023 01:40
Para: community/community
Cc:Betezek; Author
Assunto: Re: [community/community] dificuldade em entender este processo dedivulgar um app (Discussion #70070)

O GitHub não serve como meio publicitário, em termos de fazer com que mais pessoas vejam é usar redes sociais, blogs, forums etc.

**Reply by DJLink** [reply, marked as answer]
ah, ao dizer "fazer com que mais pessoas vejam" parecia divulgar. Em termos de fazer publisher de uma aplicacao, se for webbased, certos sites conseguem correr directamente no github. https://pages.github.com/
Se for uma aplicacao mobile pode usar Github actions para automatizar coisas desde compilar a aplicacao, fazer o executavel, e depois só tem de fazer publish na App Store ou Google Play https://docs.github.com/en/actions

espero que isto ajude.

**Comment by Betezek** [comment]
Vou estudar com muita atenção. Agradeço sua gentileza de me orientar.Cordialmente Fernando Em 15 de out. de 2023 03:39, David Amador ***@***.***> escreveu:
ah, ao dizer "fazer com que mais pessoas vejam" parecia divulgar. Em termos de fazer publisher de uma aplicacao, se for webbased, certos sites conseguem correr directamente no github. https://pages.github.com/
Se for uma aplicacao mobile pode usar Github actions para automatizar coisas desde compilar a aplicacao, fazer o executavel, e depois só tem de fazer publish na App Store ou Google Play https://docs.github.com/en/actions
espero que isto ajude.

**Comment by gaganachiranjitha99** [comment]
test1

**Reply by KoshikaDul1999** [reply]
khj

**Comment by ghostinhershell** [comment]
Hi @Betezek, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
